Homework_22/script.js

```diff
@@ -0,0 +1,26 @@
+let first = +prompt("Enter first number");
+let second = +prompt("Enter second number");
+
+let a = first + second;
+let b = first - second;
+let c = first / second;
+let d = first * second;
+
+if (second === 0) {
+  alert("Error");
+} else {
+  alert(c);
+}
+
+if (first < second) {
+  let boolean = confirm("Are you sure you want to carry on with calculation?");
+  if (boolean === true) {
+    alert(b);
+  }
+}
+
+if ((second === 0 || null || undefined) || (first === 0 ||null || undefined)) {
```

Comment:
1. `0` is quite valid value except for division, why not to pass it?
2. you don't have checking if user pass some string, what's then?

Reply:
3. this checking doesn't work like that, you have to repeat the left side for every checking, I mean:
`(second === 0 || second === null || second === undefined)`

Reply:
4. better to use checking like I showed on the last leacture:
```
if (typeof second !== 'number' || typeof first !== 'number') {
  alert("Error");
} else {
   alert(a + " " + b + " " + c + " " + d);
 }
```